apps/web/src/server/appeals/national-list/__tests__/national-list.test.js

```diff
@@ -13,8 +13,30 @@ describe('national-list', () => {
 	afterEach(teardown);
 
 	describe('GET /', () => {
-		it('should render national list', async () => {
-			nock('http://test/').get('/appeals/').reply(200, appealsNationalList);
+		it('should render national list - no pagination', async () => {
+			nock('http://test/').get('/appeals?pageNumber=1&pageSize=30').reply(200, appealsNationalList);
```

Comment:
As this test is for no pagination, it might be clearer to someone reading in the future if the url didn't include the `pageNumber` and `pageSize` params.

It is just a readability thing though as it makes no difference to the functionality.